solarforecastarbiter/validation/validator.py

```diff
@@ -59,13 +59,12 @@ def _QCRad_ub(dni_extra, sza, lim):
     return lim['mult'] * dni_extra * cosd(sza)**lim['exp'] + lim['min']
 
 
-def check_irradiance_limits_QCRad(irrad, test_dhi=False, test_dni=False,
-                                  limits=None):
+def check_irradiance_limits_QCRad(ghi, solar_zenith, dni_extra, dhi=None,
```

Comment:
The code for each component is independent, so perhaps a different function for each of ghi, dhi, and dni? 

`check_ghi_limits_QCRad`, `check_dni_limits_QCRad`, `check_dhi_limits_QCRad`. Could also keep `check_irradiance_limits_QCRad` as a wrapper around all of them.